build.rs

```diff
@@ -7,10 +7,19 @@ fn main() -> Result<()> {
 
     println!("cargo:rerun-if-changed=package.json");
     println!("cargo:rerun-if-changed=package-lock.json");
-    let status = Command::new("npm").arg("install").spawn()?.wait()?;
-    if !status.success() {
-        std::process::exit(1);
+    match Command::new("npm").arg("install").spawn() {
+        Err(_proc) => {
+            println!("Build requires npm, but it was not found. Please install Node >= 16.16.0.");
+            std::process::exit(1);
+        }
+        Ok(mut child) => {
+            let status = child.wait()?;
+            if !status.success() {
+                std::process::exit(1);
+            }
```

Comment:
What does it look like if this case is it? The process' stderr is still printed, right? I just want to make sure we don't have a mysterious termination without being able to figure out what went wrong.

Still might want to have aperf print a line stating that the command exited unsuccessfully (and provide the command that failed).

Reply:
Added